ampligraph/evaluation/protocol.py

```diff
@@ -482,9 +482,11 @@ def evaluate_performance(X, model, filter_triples=None, verbose=False, strict=Tr
 
     verbose : bool
         Verbose mode
-    strict : bool
-        Strict mode. If True then any unseen entity will cause a RuntimeError.
-        If False then triples containing unseen entities will be filtered out.
+    strict : int
```

Comment:
Change strict to 'filter_unseen': { True, False}. 

Give log of number of unseen entities filtered out, or a warning indicating that unseen entities will cause a tensorflow runtime error if unhandled.